### PR TITLE
Validation: Added Preserve Views

### PR DESCRIPTION
**Pull request by dereklh7**
Added preserve views to various links.

### CODE REVIEW

File: concepts/onenote-update-page.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "Update OneNote page content"
-description: " Enterprise notebooks on Microsoft 365"
+description: "Describes how to update OneNote page content and outlines how to construct the request URI and message body, and provides example requests."
```
Comment (brittmsantos) [inline concepts/onenote-update-page.md:3]: I'd consider changing the "and" in "content and outlines" to a comma, e.g. "Describes how to update OneNote page content, outlines how to construct the request URI and message body, and provides example requests."

**Comment by dereklh7** [comment]
@Lauragra @Danielabom 

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

**Comment by dereklh7** [comment]
@Lauragra @Danielabom I've resolved merge conflicts for this PR. Please let me know if there are any remaining issues preventing approval and merge for this PR. Thanks!

File: concepts/onenote-input-output-html.md [hunk 2]
```diff
@@ -63,7 +63,7 @@ The OneNote APIs in Microsoft Graph wrap all body content in at least one div. T
 |Output attribute|Description|
 |:------|:------|
 | data-id | A reference for the element.<br/><br/>Used to [update page content](onenote-update-page.md). |
-| id | A unique, generated ID for the element. Returned by [GET requests to a page's *content* endpoint](/graph/api/page-get) when the `includeIDs=true` query option is used.<br/><br/>Used to [update page content](onenote-update-page.md). |
+| id | A unique, generated ID for the element. Returned by [GET requests to a page's *content* endpoint](/graph/api/page-get?view=graph-rest-1.0&preserve-view=true) when the `includeIDs=true` query option is used.<br/><br/>Used to [update page content](onenote-update-page.md). |
```
Comment (Lauragra) [inline concepts/onenote-input-output-html.md:66]: Global: In topics in the concepts folder, we've been removing the version parameter from the links so that we always default to the latest version, in the event that we release a version 2.0 in the future. Instead of adding the preserve parameter, can you please remove the version and the preserve parameters?

File: api-reference/beta/api/identityproviderbase-update.md [hunk 3]
```diff
@@ -97,12 +97,9 @@ If successful, this method returns a `204 No Content` response code. If unsucces
 The following is an example of the request.
 
 
-# [HTTP](#tab/http)
```
Comment (Lauragra) [inline api-reference/beta/api/identityproviderbase-update.md:100]: Why did you remove the HTTP tab from this topic? We should have an HTTP tab.

File: api-reference/beta/api/identityproviderbase-update.md [hunk 4]
```diff
@@ -97,12 +97,9 @@ If successful, this method returns a `204 No Content` response code. If unsucces
...
-<!-- {
-  "blockType": "request",
-  "name": "update_socialidentityprovider"
-}
--->
+# [C#](#tab/csharp)
+[!INCLUDE [sample-code](../includes/snippets/csharp/update-socialidentityprovider-csharp-snippets.md)]
```
Comment (Lauragra) [inline api-reference/beta/api/identityproviderbase-update.md:101]: Is this the correct c# snippet? It doesn't look like it. If a snippet doesn't exist for a particular language, we link to this includes file: https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/includes/snippets/snippet-unavailable.md

**Comment by dereklh7** [comment]
@Lauragra I have implemented requested fixes. Please let me know if there are any additional issues with this PR preventing approval and merge. Thanks!

File: api-reference/beta/api/identityproviderbase-update.md [hunk 5]
```diff
@@ -105,16 +105,15 @@ The following is an example of the request.
 -->
 
 ``` http
-PATCH https://graph.microsoft.com/beta/identity/identityProviders/Amazon-OAUTH
+PATCH https://graph.microsoft.com/v1.0/identity/identityProviders/Amazon-OAUTH
```
Comment (Lauragra) [inline api-reference/beta/api/identityproviderbase-update.md:108]: Why was this changed to v1.0? This is a beta topic and should have /beta in the example requests.

File: concepts/onenote-input-output-html.md [hunk 6]
```diff
@@ -120,7 +120,7 @@ Images on OneNote pages are represented by **img** elements. An **img** element
...
 | data-id | A reference for the element.<br/><br/>Used to [update page content](onenote-update-page.md). |
-| data-render-src |Either **data-render-src** or **src** is required.<br/><br/>The webpage to render as a bit-mapped image on the OneNote page:<br/><br/> - `data-render-src="https://..."` for a public URL.<br/><br/> - `data-render-src="name:BlockName"` for an image part in the "Presentation" block of a [multipart request](/graph/api/section-post-pages#example).<br/><br/>This method is useful when the webpage is more complex than the OneNote page can faithfully render, or when the page requires login credentials.|
+| data-render-src |Either **data-render-src** or **src** is required.<br/><br/>The webpage to render as a bit-mapped image on the OneNote page:<br/><br/> - `data-render-src="https://..."` for a public URL.<br/><br/> - `data-render-src="name:BlockName"` for an image part in the "Presentation" block of a [multipart request](/graph/api/section-post-pages?view=graph-rest-1.0#example&preserve-view=true).<br/><br/>This method is useful when the webpage is more complex than the OneNote page can faithfully render, or when the page requires login credentials.|
```
Comment (Lauragra) [inline concepts/onenote-input-output-html.md:123]: Global: In all concept topics, please remove the version parameter and the preserve parameter. We should not be linking to a specific version of reference content in concept topics; the link will default to the latest version, We have a vendor doing a scrub to remove this from all topic so it's possible that this work has already been done. Please be sure to update your branch to prevent possible merge conflicts.

**Comment by dereklh7** [comment]
@Lauragra I have made the requested adjustments. Please let me know if there are any additional adjustments needed for this PR to be approved and merged. Thanks and have a great day!

File: concepts/onenote-input-output-html.md [hunk 7]
```diff
@@ -143,15 +142,15 @@ Images on OneNote pages are represented by **img** elements. An **img** element
 | data-render-original-src | The original source URL of the image, if the source image is from the public internet and was created with the **data-render-src** attribute. |
 | data-src-type | The media type of the **src** resource, for example: `image/png` or `image/jpeg`. |
 | data-tag | A [note tag](onenote-note-tags.md) on the element. |
-| id | A unique, generated ID for the element. Returned by [GET requests to a page's *content* endpoint](/graph/api/page-get) when the `includeIDs=true` query option is used.<br/><br/>Used to [update page content](onenote-update-page.md). |
+| id | A unique, generated ID for the element. Returned by [GET requests to a page's *content* endpoint](/graph/api/page-get?view=graph-rest-1.0&preserve-view=true) when the `includeIDs=true` query option is used.<br/><br/>Used to [update page content](onenote-update-page.md). |
```
Comment (Lauragra) [inline concepts/onenote-input-output-html.md:145]: Global: For all topics in the concepts folder, please remove this from the URLs: ?view=graph-rest-1.0&preserve-view=true. We are doing a pass to remove the versioned links so the links will automatically default to the latest version. This PR is adding them back when we are intentionally removing them. Thanks.

**Comment by dereklh7** [comment]
@Lauragra I've resolved merge conflicts and implemented requested fixes. Please let me know if there are any additional issues with this PR that need to be resolved for this PR to be approved and merged. Thanks!

File: concepts/onenote-input-output-html.md [hunk 8]
```diff
@@ -121,7 +121,6 @@ Images on OneNote pages are represented by **img** elements. An **img** element
 | alt | The supplied alt text for the image. |
 | data-id | A reference for the element.<br/><br/>Used to [update page content](onenote-update-page.md). |
 | data-render-src |Either **data-render-src** or **src** is required.<br/><br/>The webpage to render as a bit-mapped image on the OneNote page:<br/><br/> - `data-render-src="https://..."` for a public URL.<br/><br/> - `data-render-src="name:BlockName"` for an image part in the "Presentation" block of a [multipart request](/graph/api/section-post-pages#example).<br/><br/>This method is useful when the webpage is more complex than the OneNote page can faithfully render, or when the page requires login credentials.|
-| data-tag | A [note tag](onenote-note-tags.md) on the element. |
```
Comment (Lauragra) [inline concepts/onenote-input-output-html.md:124]: Why was this attribute deleted?

File: concepts/onenote-input-output-html.md [hunk 9]
```diff
@@ -588,7 +587,7 @@ This is the input HTML sent in the message body to create the page.
 
 This is the output HTML that Microsoft Graph returns when you [get page content](onenote-get-content.md).
 
-> [!NOTE]
```
Comment (Lauragra) [inline concepts/onenote-input-output-html.md:591]: Why was the note syntax changed here?

File: concepts/onenote-note-tags.md [hunk 10]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Use note tags on OneNote pages by using the OneNote API"
+title: "Use note tags in OneNote pages"
```
Comment (Lauragra) [inline concepts/onenote-note-tags.md:2]: I'm not sure why this change; our vendor editor had changed this as part of an editorial/SEO pass.

File: concepts/outlook-calendar-concept-overview.md [hunk 11]
```diff
@@ -84,9 +84,9 @@ Depending on your app scenario, you can use the places API within the context of
 
 ## Take advantage of social intelligence and other developer conveniences in Microsoft Graph
 
-Use the [people API](people-example.md) in Microsoft Graph to connect to [people data](/graph/api/resources/person), which is based on a user's communication and collaboration patterns and business relationships. You can implement controls such as a people picker and suggest persons relevant to the user when organizing meetings on the user's behalf.
+Use the [people API](people-example.md) in Microsoft Graph to connect to [people data](/graph/api/resources/person) which is based on a user's communication and collaboration patterns, and business relationships. You can implement controls such as a people picker, and suggest persons relevant to the user when organizing meetings on the user's behalf.
```
Comment (Lauragra) [inline concepts/outlook-calendar-concept-overview.md:87]: Please remove the comma here, those are two concepts so we shouldn't separate with a comma.

File: concepts/outlook-get-mime-message.md [hunk 12]
```diff
@@ -42,7 +42,7 @@ creation and last modification dates.
 ## Get MIME content of an Outlook message
 
 You can get the MIME representation of a message by appending the `$value` segment when 
-[getting the message](/graph/api/message-get): 
+[getting the message](/graph/api/message-get?view=graph-rest-1.0&preserve-view=true):
```
Comment (Lauragra) [inline concepts/outlook-get-mime-message.md:45]: Why did we add the version and the preserve parameter here?

File: concepts/outlook-get-mime-message.md [hunk 13]
```diff
@@ -275,6 +274,8 @@ e.</p>
 
 ## Next steps
 
+Find out more about:
+
 - [Get the MIME content of an item attachment](/graph/api/attachment-get#get-the-raw-contents-of-a-file-or-item-attachment) to an event, message, Outlook task, or group post
 - [Why integrate with Outlook mail](outlook-mail-concept-overview.md)
-- [Use the mail API](/graph/api/resources/mail-api-overview) and its [use cases](/graph/api/resources/mail-api-overview#common-use-cases) in Microsoft Graph v1.0
+- [Using the mail API](/graph/api/resources/mail-api-overview) and its [use cases](/graph/api/resources/mail-api-overview#common-use-cases) in Microsoft Graph v1.0
```
Comment (Lauragra) [inline concepts/outlook-get-mime-message.md:281]: Please restore "Use" here to match the topic title.

**Comment by dereklh7** [comment]
@Lauragra I have implemented requested changes. Please let me know if there are any outstanding issues with this PR preventing approval and merge. Thanks and have a great day!

**Review by Lauragra — APPROVED** [review]
